### ISSUE TITLE
Missing UseMySQL(string connectionString, Action<MySQLDbContextOptionsBuilder> MySQLOptionsAction) extension overload

### ISSUE THREAD

**Issue by SapientGuardian** [SapientGuardian/SapientGuardian.EntityFrameworkCore.MySql#1]
The MSSQL, PGSQL, and Sqlite EF providers all have a Use_(string connectionString, Action<_DbContextOptionsBuilder> *OptionsAction) extension method, which seems to be useful for migrations.

Here are some examples, borrowed from https://github.com/damienbod/AspNet5MultipleProject (which I'm trying to make work with MySql)

```
// Use a SQLite database
            //var sqlConnectionString = Configuration["DataAccessSqliteProvider:ConnectionString"];

            //services.AddDbContext<DomainModelSqliteContext>(options =>
            //    options.UseSqlite(
            //        sqlConnectionString,
            //        b => b.MigrationsAssembly("AspNet5MultipleProject")
            //    )
            //);

            //services.AddScoped<IDataAccessProvider, DataAccessSqliteProvider.DataAccessSqliteProvider>();

            // Use a MS SQL Server database
            //var sqlConnectionString = Configuration["DataAccessMsSqlServerProvider:ConnectionString"];

            //services.AddDbContext<DomainModelMsSqlServerContext>(options =>
            //    options.UseSqlServer(
            //        sqlConnectionString,
            //        b => b.MigrationsAssembly("AspNet5MultipleProject")
            //    )
            //);

            //services.AddScoped<IDataAccessProvider, DataAccessMsSqlServerProvider.DataAccessMsSqlServerProvider>();

            //Use a PostgreSQL database
            var sqlConnectionString = Configuration["DataAccessPostgreSqlProvider:ConnectionString"];

            services.AddDbContext<DomainModelPostgreSqlContext>(options =>
                options.UseNpgsql(
                    sqlConnectionString,
                    b => b.MigrationsAssembly("AspNet5MultipleProject")
                )
            );

            //services.AddScoped<IDataAccessProvider, DataAccessPostgreSqlProvider.DataAccessPostgreSqlProvider>();
```

The MySql provider only has UseMySQL extensions that take a connection or connection string, but not the action parameter. The strange thing is that I can't find the code for those extensions in any of the other libraries! 

http://sourcebrowser.io/Browse/aspnet/EntityFramework/src/EntityFramework.SqlServer/Extensions/SqlServerDbContextOptionsExtensions.cs
https://raw.githubusercontent.com/borgdylan/npgsql-ef7/master/EntityFramework.Npgsql/Extensions/NpgsqlDbContextOptionsExtensions.cs

**Comment by SapientGuardian** [comment]
@how02 can you take a look?

**Comment by how02** [comment]
Done, I've created a pull request.

**Comment by SapientGuardian** [comment]
Fixed in #2
